addons/account/models/account_account_tag.py

```diff
@@ -20,7 +20,7 @@ def _get_tax_tags(self, tag_name, country_id):
         """ Returns all the tax tags corresponding to the tag name given in parameter
         in the specified country.
         """
-        escaped_tag_name = tag_name.replace('\\', '\\\\').replace('%', '\%').replace('_', '\_')
+        escaped_tag_name = tag_name.replace('\\', '\\\\').replace('%', r'\%').replace('_', r'\_')
```

Comment:
```suggestion
        escaped_tag_name = tag_name.replace('\\', '\\\\').replace('%', '\\%').replace('_', '\\_')
```
Shouldn't this be done like this? Those are not really regex strings, they are just wrongly escaped no?
It's doing the same with double `\` but prevent the linter error
```py
>>> "a_b".replace('_', '\_')
'a\\_b'
>>> "a_b".replace('_', '\\_')
'a\\_b'
```